### PR TITLE
Add unit tests

### PR DESCRIPTION
**Pull request by esheldon**

### CODE REVIEW

File: meds/defaults.py [hunk 1]
```diff
@@ -1,15 +1,15 @@
-__version__="0.9.12"
+__version__ = "0.9.12"
```
Comment (beckermr) [inline meds/defaults.py:1]: I missed changing this to 0.9.13. :(

File: meds/meds.py [hunk 2]
```diff
@@ -1042,24 +1030,11 @@ def _make_composite_image(
...
-        elif type == "ormask":
-            return "ormask_cutouts"
-        elif type == "noise":
-            return "noise_cutouts"
-        else:
-            ext = "%s_cutouts" % type
+        ext = "%s_cutouts" % type
+        if ext not in self._fits:
             if ext not in self._fits:
                 raise ValueError("bad cutout type '%s'" % type)
```
Comment (beckermr) [inline meds/meds.py:1036]:
```suggestion
            raise ValueError("bad cutout type '%s'" % type)
```

Why the extra if statement here?

**Review by beckermr — COMMENTED** [review]
A few comments. Also 

- This extra file got committed `meds/tests/.coverage`. Maybe add to the .gitignore?
- The extractors do not support extra extenions. This is probably fine. If it isn't, then I can add that.

**Comment by esheldon** [comment]
shall we merge this and add more tests later?

**Review by beckermr — APPROVED** [review]

**Pull request merged**